src/tasks/Promotions/YouTube/Routes.ts

```diff
@@ -0,0 +1,64 @@
+import * as express from "express";
+import { channelIds } from "../../../channel-IDs";
+
+export const youtube = express.Router();
+
+youtube.get('/notification', ({ query: { 'hub.challenge': challenge } }, res) => {
+  res.status(200).end(challenge);
+});
+
+youtube.post('/notification', ({ body, app }, res) => {
+  try {
+    const publications = body.feed.entry;
+    
+    if (!publications) {
+      throw new Error('Request without publication');
+    };
+    
+    const discordClient = app.get('discordClient');
```

Comment:
You can use type assertion to set the type.  And could also verify to make sure it's a discord client:

```javascript
const discordClient = app.get('discordClient') as Client;

if (!(discordClient instanceof Client)) throw...
```